Hardware/Hardware/CardReader/StripeCardReader/StripeCardReaderService.swift

```diff
@@ -447,8 +457,6 @@ extension StripeCardReaderService: DiscoveryDelegate {
 // MARK: - ReaderDisplayDelegate.
 extension StripeCardReaderService: BluetoothReaderDelegate {
     public func reader(_ reader: Reader, didReportAvailableUpdate update: ReaderSoftwareUpdate) {
-        let softwareUpdate = CardReaderSoftwareUpdate(update: update)
```

Comment:
Are we sure this is safe to remove? How would we be notified that there is an update available?

Reply:
I think so. Before, reader updates were notified through the `readerEvents` publisher, but now they have a dedicated `softwareUpdateEvents` publisher. When I updated the code to handle optional updates, the previous method got unused, so I cleaned up all the related stuff.

I had doubts about removing the `CardReaderSoftwareUpdate`, but since we're not showing any details about the software update version in the UI, that's unused as well

Reply:
Ah, I see. I misread the diff, sorry. I thought we would be missing on available updates. 

> I had doubts about removing the CardReaderSoftwareUpdate, but since we're not showing any details about the software update version in the UI, that's unused as well

That's a good point. I would lean towards not removing it though, it seems like it's information that we might want to know later on, maybe attach to a Tracks event? Anyway, not a blocker. :shipit:

Reply:
I thought about that, but we can always revert/restore that code if we need it